v2/dtos/reading.go

```diff
@@ -18,12 +18,11 @@ import (
 type BaseReading struct {
 	common.Versionable `json:",inline"`
 	Id                 string   `json:"id"`
-	Pushed             int64    `json:"pushed,omitempty"`
 	Created            int64    `json:"created"`
 	Origin             int64    `json:"origin"`
```

Comment:
I think  `Origin` should be required?

Reply:
Why does `Modified` have "omitempty" but `Created` does not?

Reply:
Updated `Origin` as required in `BaseReading` DTO.